Lib/test/test_dis.py

```diff
@@ -909,6 +909,14 @@ def test_widths(self):
     def test_dis(self):
         self.do_disassembly_test(_f, dis_f)
 
+    def test_disco(self):
+        func = _f
+        disassemble_output = self.get_disassembly(func.__code__, wrapper=False)
+        disco_output = io.StringIO()
+        with contextlib.redirect_stdout(disco_output):
+            dis.disco(func.__code__)
+        self.assertEqual(disassemble_output, disco_output.getvalue())
+
```

Comment:
We could, instead of adding this test, add a check in ``do_disassembly_test`` that disco gives the expected output as well.  That would cover more cases.

Reply:
@iritkatriel I've updated the PR to reflect your suggestion. Please let me know if you have any other suggestions.